config/initializers/rollbar.rb

```diff
@@ -0,0 +1,85 @@
+require 'rollbar/rails'
+
+Rollbar.configure do |config|
+  # Without configuration, Rollbar is enabled in all environments.
+  # To disable in specific environments, set config.enabled=false.
+
+  config.access_token = ENV['ROLLBAR_ACCESS_TOKEN']
+
+  # Here we'll disable in 'test' and dev:
+  if !Rails.env.production?
+    config.enabled = false
+  end
+
+  config.environment = ENV["APP_ENVIRONMENT"] || "development"
+
+  # By default, Rollbar will try to call the `current_user` controller method
```

Comment:
do you remind removing all of these comments? I find it easier to read these config files if they are free of documentation comments

Reply:
Yep. These came from another add-on service that I suspect used the templated Rollbar config file.